File: server/src/Bananagrams.Dal/Models/Game.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Bananagrams.Dal.Models;

[Table("games")]
public class Game
{
    [Key]
    [Column("id")]
    public int Id { get; set; }
    [Column("title")]
    public string Title { get; set; }    
    [Column("date_created")]
    public DateTime DateCreated { get; set; }
    public List<GameUser>? GameUsers { get; set; }
    public List<GameAnagram>? GameAnagrams { get; set; }
}

File: server.java/bananagrams/src/main/java/coe/bananagrams/entity/Game.java
package coe.bananagrams.entity;

import jakarta.persistence.*;
import lombok.*;

import java.util.Date;
import java.util.Set;

@Getter
@Setter
@Entity
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "games")
public class Game {

    @Id
    @Column(name = "id", updatable = false, nullable = false)
    private int id;

    @Column(name = "title", nullable = false)
    private String title;

    @Column(name = "date_created", nullable = false)
    private Date dateCreated;

    @OneToMany(mappedBy = "game")
    private Set<GameAnagram> gameAnagrams;
    @OneToMany(mappedBy = "game")
    private Set<GameUser> gameUsers;

}
